configs/beit/upernet_beit_base_12_640_slide_160k_ade20k_ms.py

```diff
@@ -0,0 +1,70 @@
+_base_ = [
+    '../_base_/models/upernet_beit.py', '../_base_/datasets/ade20k_640x640.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_160k.py'
+]
+crop_size = (640, 640)
+
+model = dict(
+    pretrained='pretrain/beit_base_patch16_224_pt22k_ft22k_new1.pth',
+    backbone=dict(
+        type='BEiT',
+        img_size=(640, 640),
+        patch_size=16,
+        embed_dims=768,
+        num_layers=12,
+        num_heads=12,
+        mlp_ratio=4,
+        qv_bias=True,
+        init_values=0.1,
+        drop_path_rate=0.1,
+        out_indices=[3, 5, 7, 11]),
+    neck=dict(embed_dim=768, rescales=[4, 2, 1, 0.5]),
+    decode_head=dict(
+        in_channels=[768, 768, 768, 768], num_classes=150, channels=768),
+    auxiliary_head=dict(in_channels=768, num_classes=150),
+    test_cfg=dict(mode='slide', crop_size=crop_size, stride=(426, 426)))
+
+optimizer = dict(
+    _delete_=True,
+    type='AdamW',
+    lr=3e-5,
+    betas=(0.9, 0.999),
+    weight_decay=0.05,
+    constructor='LayerDecayOptimizerConstructor',
+    paramwise_cfg=dict(num_layers=12, layer_decay_rate=0.9))
+
+lr_config = dict(
+    _delete_=True,
+    policy='poly',
+    warmup='linear',
+    warmup_iters=1500,
+    warmup_ratio=1e-6,
+    power=1.0,
+    min_lr=0.0,
+    by_epoch=False)
+
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+min_size = 640
+find_unused_parameters = True
```

Comment:
Is this necessary?

Reply:
`find_unused_parameters` may not be kept in MMSegmentation, because our default setting in DDP is `find_unused_parameters=False` , and it is not recommanded to set it `True` because it means some parameters would not be used. For example, check out the bottom of [STDC](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/stdc).